vpr/src/route/route_timing.cpp

```diff
@@ -1960,6 +1960,16 @@ void Router::timing_driven_expand_neighbours(t_heap* current,
     RREdgeId first_edge = rr_nodes_->first_edge(from_node);
     RREdgeId last_edge = rr_nodes_->last_edge(from_node);
     int num_edges = size_t(last_edge) - size_t(first_edge);
+    for (int iconn = 0; iconn < num_edges; iconn++) {
+        RREdgeId from_edge(size_t(first_edge) + iconn);
+
+        RRNodeId to_node = rr_nodes_->edge_sink_node(from_edge);
```

Comment:
Explain why you're prefetching (to_node and rr_switch_inf will be used soon) and what speed up you got.